src/linked_list.py

```
@@ -4,10 +4,16 @@
 class LinkedList(object):
     """Class representation of linked list."""
 
-    def __init__(self, head_node):
+    def __init__(self, iterable=None):
         """Instantiate linked list."""
-        self.head_node = head_node
-        self.length = 1
+        self.head_node = None
+        self.length = 0
+        try:
```

Very good! You probably want to go ahead and raise an exception instead of just printing a message. Raising an exception is the best way to let users know that they done fucked up.